nox/_typing.py

```diff
@@ -14,33 +14,8 @@
 
 from __future__ import annotations
 
-__all__ = ["TYPE_CHECKING", "ClassVar", "NoReturn", "Python"]
```

Comment:
TYPE_CHECKING was added in 3.5.2, ClassVar in 3.5.3, and NoReturn in 3.5.4/3.6.2. All are now always present.